pkg/management/component.go

```diff
@@ -316,13 +316,22 @@ func (c *component) GetRequiredActions(ctx context.Context, realmName string) ([
 	return requiredActionsRep, nil
 }
 
-func (c *component) CreateUser(ctx context.Context, realmName string, user api.UserRepresentation, generateUsername bool, generateNameID bool) (string, error) {
+func (c *component) CreateUser(ctx context.Context, realmName string, user api.UserRepresentation, generateUsername bool, generateNameID bool, termsOfUse bool) (string, error) {
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 	var ctxRealm = ctx.Value(cs.CtContextRealm).(string)
 
 	var userRep = api.ConvertToKCUser(user)
 	userRep.SetAttributeString(constants.AttrbSource, "api")
 
+	if termsOfUse {
+		var reqActions []string
+		if userRep.RequiredActions != nil {
```

Comment:
I don't see how it could happen

Reply:
Currently, it's not possible... but if someone sets the RequiredActions field with one or more required actions in api.ConvertToKCUser(...), he might not see that his required actions might be ignored as in most cases, he won't set termsOfUse=true

Reply:
ok